modules/ldap-syncing-activedir.adoc

```diff
@@ -48,6 +48,9 @@ during search and returned to the client, but not committed to the database.
 .Prerequisites
 
 * Create the configuration file.
+ifdef::openshift-dedicated,openshift-rosa[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-dedicated,openshift-rosa[]
```

Comment:
The next 9 files, you add the dedicated-admin prereq. Should you also add  a cluster-admin prereq for `ifndef:openshift-dedicated,openshift-rosa`?

Reply:
@bergerhoffer I'm happy to make this change to the OCP content if you agree. For all of the Authentication procedures, I added a `dedicated-admin` prereq as needed. I assume that we need a corresponding `cluster-admin` prereq for OCP.

Reply:
@bhardesty Yes that looks like it would be appropriate since they all are `oc adm` commands. Thanks!

I will try to make time to review this PR tomorrow.

Reply:
Done! I added the cluster-admin prereqs for OCP.